phpstan.neon

```diff
@@ -2,13 +2,17 @@ parameters:
     paths:
         - ../../../app
     excludePaths:
+        - ./../../app/App/Console/Playbooks
+        - ./../../app/App/Nova/*
     level: 8
     ignoreErrors:
         - '#is not allowed to extend#'
         - '#Constructor in ([\w\\]+) has parameter ([\$\w]+) with default value.#'
         - '#Method ([\w\\\:_\(\)]+) has parameter ([\$\w]+) with a nullable type declaration.#'
         - '#Method ([\w\\\:_\(\)]+) has parameter ([\$\w]+) with null as default value.#'
-        - '#Method ([\w\\\:_\(\)]+) has a nullable return type declaration.#'
+        - '#has a nullable return type declaration.#'
         - '#Dynamic call to static method Illuminate\\Database\\Eloquent\\((Builder|Relations|Model)+[\w\\\:\(\)\<\>]+).#'
+        - '#Method [a-zA-Z0-9\\_]+\\Components\\[a-zA-Z0-9\\_]+\:\:rules\(\) is protected, but since the containing class is final, it can be private#'
+        - '#Method [a-zA-Z0-9\\_]+\\Livewire\\[a-zA-Z0-9\\_]+\:\:rules\(\) is protected, but since the containing class is final, it can be private#'
```

Comment:
@alfonsobries Doesn't it work something like this?
```suggestion
        - '#Method ([\w\\]+(Livewire|Components)+[\w\\\:\(\)\<\>]+\:\:rules\(\)) is protected, but since the containing class is final, it can be private.#'
```

Reply:
@leMaur yep, nice 🌮 incoming